ci/docker/base.test.Dockerfile

```diff
@@ -64,4 +58,4 @@ WORKDIR /ray
 # Below should be re-run each time
 COPY . .
 
-RUN ./ci/env/install-dependencies.sh init
+RUN CI=true ./ci/env/install-bazel.sh
```

Comment:
Does this make install-dependencies slower on non-base build?

Reply:
this step (`install-deps init`) does not install python 3rd party deps.

the things that `init` does today:
- install/upgrade bazel
- install/upgrade pip
- (re)install the `thirdparty_files` stuff

the larger part of python dependencies are installed in later layers, and pip and third_party files stuff are going to be reinstalled later by `ci/ci.sh init` anyways.